src/controllers/transaction.controller.ts

```diff
@@ -65,27 +57,22 @@ export class TransactionController {
     const {from, to, value, cryptocurrencyId: cryptoId, postId} = transaction;
 
     // Validate if crypto already exist
-    await this.cryptocurrencyRepository.findById(
-      transaction.cryptocurrencyId.toUpperCase(),
-    );
+    await this.cryptocurrencyRepository.findById(transaction.cryptocurrencyId.toUpperCase());
```

Comment:
use camelCase
from `cryptocurrencyRepository` to `cryptoCurrencyRepository`

Reply:
cryptocurrency bukannya 1 kata ya, bukan 2 kata yang terpisah